src/app/modules/main/wallet_section/filter.nim

```diff
@@ -0,0 +1,44 @@
+import strformat, sequtils, sugar
+
+import ./controller
+
+
+type Filter* = ref object
+  controller: Controller
+  addresses*: seq[string]
```

Comment:
@alaibe does this need to be `addresses*: seq[string]` or `address*: string` (singular) does the thing as well, in the code below I don't see the need for having this as an array.

Reply:
For the combined view, we need this addresses list, also for we can remove watch only account, that is why it is the list. I am yet to implement the services part that leverage this